ironic/templates/configmaps/nginx-default.conf.yaml.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  default.conf: |+
```

Comment:
I would also redirect access and error log to std{out,err}:
error_log /dev/fd/2 info;

http {
  access_log /dev/fd/1;
  ...
}